src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -924,10 +924,15 @@ internal override bool HasSpecialName
             }
         }
 
-        public override bool NonNullTypes
+        public override bool? NonNullTypes
         {
             get
             {
+                if (!DeclaringCompilation.IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking))
+                {
+                    return false;
+                }
```

Comment:
Members (methods, properties, etc.) are not checking this value. What is different about types?

Reply:
That's an oversight. They should all check. Thanks

---
In reply to: [205842585](https://github.com/dotnet/roslyn/pull/28825#discussion_r205842585) [](ancestors = 205842585)